pkg/daemon/ceph/osd/kms/vault_api.go

```diff
@@ -33,6 +33,9 @@ const (
 	kvVersion2   = "kv-v2"
 )
 
+// VaultClient returns a vault client, also used in unit tests to mock the client
+var VaultClient = newVaultClient
```

Comment:
looks like this is only used in the package?
```suggestion
var vaultClient = newVaultClient
```